robottelo/host_helpers/system_tools.py

```diff
@@ -0,0 +1,92 @@
+import contextlib
+import random
+
+from robottelo.cli.proxy import CapsuleTunnelError
+from robottelo.config import settings
+from robottelo.logging import logger
+
+
+def get_available_capsule_port(capsule, port_pool=None):
```

Comment:
since both of these functions deal with capsules, how about this file is renamed `capsule_mixins.py` and is nested under a mixin class. This will allow the functions to work without needing a capsule class passed in.

Reply:
renamed and nested